app/uk.gov.hmrc.helptosavestridefrontend/views/partials/link.scala.html

```diff
@@ -0,0 +1,39 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(
+  key: String,
+  href:String,
+  classes: Option[String] = None,
+  rel: Option[String] = None,
+  target: Option[String] = None,
+  id: Option[String] = None,
+  attrs: Map[String, String] = Map.empty
+ )(implicit msgs: Messages)
+
+@cls = @{classes.getOrElse("govuk-link govuk-body")}
```

Comment:
links should not have govuk-body style by default https://design-system.service.gov.uk/styles/typography/#links

Reply:
done